spec/acceptance/disable_new_tasking_system_spec.rb

```diff
@@ -0,0 +1,141 @@
+require 'spec_helper_acceptance'
+
+describe 'initial configuration with new tasking system' do
+  certdir = '/etc/pulpcore-certs'
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      class { 'pulpcore':
+        worker_count              => 1,
+        use_legacy_tasking_system => false,
+      }
+      PUPPET
+    end
+  end
+
+  describe service('httpd') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-api') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-content') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-resource-manager') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-worker@1') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe port(80) do
+    it { is_expected.to be_listening }
+  end
+
+  describe port(443) do
+    it { is_expected.to be_listening }
+  end
+
+  describe curl_command("https://#{host_inventory['fqdn']}/pulp/api/v3/status/", cacert: "#{certdir}/ca-cert.pem") do
+    its(:response_code) { is_expected.to eq(200) }
+    its(:exit_status) { is_expected.to eq 0 }
+  end
+
+  describe command("PULP_SETTINGS=/etc/pulp/settings.py pulpcore-manager diffsettings") do
```

Comment:
This is somewhat of a hack. `diffsettings` will show the difference between default settings for Django, so pulpcore's `USE_NEW_WORKER_TYPE` setting (introduced with v3.13) should be visible here.

Reply:
Is there a reason not to just assert the contents of `settings.py`?

Reply:
I think you can also use dynaconf to dump the settings. I think that won't break when Pulp switches its default.

Reply:
> Is there a reason not to just assert the contents of `settings.py`?

To me, reading the running configuration of the application seems closer to testing the true, desired outcome vs. reading the contents of a file. For that reason I see it as a better approach in an "acceptance" test.

With that said, we could easily test both and treat one as an in memory test for the currently running process and the other as a test of the persistent configuration.

Reply:
> I think you can also use dynaconf to dump the settings. I think that won't break when Pulp switches its default.

I'll have to look into how to do this

Reply:
This test looks good enough for now. Or just yank it since we test this in the unit tests and we are asserting the behaviors we expect anyway. That's my TLDR way of saying, let's not hold this up on that sorta detail.

Reply:
> I'll have to look into how to do this

FYI: sosreport has this:
https://github.com/sosreport/sos/blob/c7d3644c0c64e9e5439806250592a55c8e2de26f/sos/report/plugins/pulpcore.py#L99-L102